modules/currency-l0/src/main/scala/org/tessellation/currency/modules/HealthChecks.scala

```diff
@@ -0,0 +1,49 @@
+package org.tessellation.currency.modules
+
+import cats.effect.Async
+import cats.effect.std.{Random, Supervisor}
+import cats.syntax.functor._
+
+import org.tessellation.currency.http.P2PClient
+import org.tessellation.kryo.KryoSerializer
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.sdk.config.types.HealthCheckConfig
+import org.tessellation.sdk.domain.cluster.services.Session
+import org.tessellation.sdk.effects.GenUUID
+import org.tessellation.sdk.infrastructure.healthcheck.ping.{PingHealthCheckConsensus, PingHealthCheckConsensusDriver}
+
+import org.http4s.client.Client
+
+object HealthChecks {
+
+  def make[F[_]: Async: KryoSerializer: GenUUID: Random: Supervisor](
+    storages: Storages[F],
+    services: Services[F],
+    programs: Programs[F],
+    p2pClient: P2PClient[F],
+    client: Client[F],
+    session: Session[F],
+    config: HealthCheckConfig,
+    selfId: PeerId
+  ): F[HealthChecks[F]] = {
+    def ping = PingHealthCheckConsensus.make(
```

Comment:
is it required? s it the local healthcheck we use currently?

Reply:
It was in `core` so I decided to just mimic it.

Reply:
`PingHealthCheckConsensus` is running on both `core` and `dag-L1`. I will extract if ofc to be DRY for now but let's discuss the local healthcheck with the rest of teeam.

Reply:
@TheMMaciek ^